string-manipulation-1/src/is-vowel.ts

```diff
@@ -0,0 +1,14 @@
+/* exported isVowel */
+function isVowel(char: string): boolean {
+  if (typeof char !== 'string' || char.length === 0) {
+    return false;
+  }
+  const lowerCaseChar = char.toLowerCase();
+  return (
+    lowerCaseChar === 'a' ||
+    lowerCaseChar === 'e' ||
+    lowerCaseChar === 'i' ||
+    lowerCaseChar === 'o' ||
+    lowerCaseChar === 'u'
+  );
```

Comment:
This function is a great example of using a `switch` statement. It almost looks like you're running a similar check as a `switch` statement but does not look as nice syntactically.
```
switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
```